speclang_modern.tex

```diff
@@ -661,6 +663,15 @@ \subsubsection{Exact computations}
 \end{listing-nonumber}
 \end{example}
 
+Note that the semantics of these constructs implies that in concept two different executions of the 
```

Comment:
This is a bit more complicated than that if the control flow might be influenced by the precision of the computation (i.e. if there are unstable tests where mathematical real arithmetic and floating point computation would give opposite answer). Indeed, execution is supposed to always follow the "normal" execution path as defined by C semantics (i.e. floating point computation): there is no such thing as a pure mathematical execution when it comes to branches.

Reply:
moved into a separate branch